src/gei/Commands/MigrateCodeScanningAlertsCommand.cs

```diff
@@ -0,0 +1,110 @@
+using System;
+using System.CommandLine;
+using Microsoft.Extensions.DependencyInjection;
+using OctoshiftCLI.Commands;
+using OctoshiftCLI.Extensions;
+using OctoshiftCLI.GithubEnterpriseImporter.Handlers;
+
+namespace OctoshiftCLI.GithubEnterpriseImporter.Commands;
+
+public class MigrateCodeScanningAlertsCommand : CommandBase<MigrateCodeScanningAlertsCommandArgs, MigrateCodeScanningAlertsCommandHandler>
+{
+    public MigrateCodeScanningAlertsCommand() : base(
+        name: "migrate-code-scanning-alerts",
+        description: "Migrates the state and resolution of code scanning alerts.")
```

Comment:
The release note contains a bit more detail on what this command does. I think it'd be a great idea to transpose that context to the help text, so it's a bit clearer what this does and what the user can expect.

Reply:
Added more context - is it better now?